src/abstractdataframe/reshape.jl

```diff
@@ -377,8 +554,10 @@ end
 
 Base.parent(v::RepeatedVector) = v.parent
 DataAPI.levels(v::RepeatedVector) = levels(parent(v))
-CategoricalArrays.isordered(v::RepeatedVector{<:Union{CategoricalValue, Missing}}) =
-    isordered(parent(v))
+
+# TODO: uncomment when DataAPI.jl supports this
```

Comment:
here we need https://github.com/JuliaData/DataAPI.jl/issues/23

Reply:
Given https://github.com/JuliaData/DataFrames.jl/pull/2494/files#r511616549 maybe we should leave this out and return `false`?